src/app/Http/Controllers/API/PresentationUserController.php

```diff
@@ -25,7 +25,20 @@ class PresentationUserController extends Controller
      */
     public function subscribe(Presentation $presentation, User $user)
     {
-        //
+        // check that the user isn't already subscribed to the presentation
+        if ($presentation->users()->find($user)) {
+            return response()->json([
+                'success' => false,
+                'message' => "Already subscribed",
+            ], 409);
```

Comment:
You should the class Response for HTTP responsees

Reply:
Done in fddd430